main/src/library/Prelude.flix

```diff
@@ -168,7 +168,7 @@ pub enum Purity3[a: Type, b: Type, c: Type, d: Type, ef: Eff] {
 /// it to be pure.
 ///
 pub def purityOf(f: a -> b \ ef): Purity[a, b, ef] = typematch f {
-    case g: a -> b \ Pure => Purity.Pure(g)
+    case g: a -> b \ {} => Purity.Pure(g)
```

Comment:
This was so nicely indented before. Can we recover that? :)

Reply:
Same for rest of file.

Reply:
I'm not sure what you mean? I only changed `Pure` -> `{}`. No indentation changes happened?

Reply:
Oh, you might mean that the `=>` align? Yep, sure I'll do that